ykbuild/src/completion_wrapper.rs

```diff
@@ -84,38 +86,35 @@ impl CompletionWrapper {
     /// Call when the build is done to generate the `compile_commands.json` file.
     pub fn generate(self) {
         let mut entries = Vec::new();
-
```

Comment:
This wasn't intentional, but I guess you are going to like it :)

Reply:
It's the best part :p